pkg/actions/addon/addon.go

```diff
@@ -36,10 +34,6 @@ type Manager struct {
 }
 
 func New(clusterConfig *api.ClusterConfig, eksAPI eksiface.EKSAPI, stackManager manager.StackManager, withOIDC bool, oidcManager *iamoidc.OpenIDConnectManager, clientSet kubeclient.Interface, timeout time.Duration) (*Manager, error) {
-	if err := supportedVersion(clusterConfig.Metadata.Version); err != nil {
```

Comment:
why are we removing this?

Reply:
Not needed, because addons are supported after 18 and now there is an earlier check which checks if we are even below that. So this check is now superseded by the fact that the minimum version supported is now supported by addons.

Reply:
noted.